app/src/main/res/values/strings.xml

```diff
@@ -696,6 +696,11 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_generalFee_title">Does it cost a fee to enter %s?</string>
     <string name="quest_laundrySelfService_title">Is this laundry a self service laundry?</string>
     <string name="quest_handrail_title">Do these steps have a handrail?</string>
+    <string name="quest_address_street_title">What street is this number %s of?</string>
+    <string name="quest_address_street_no_named_streets">This address has no named street</string>
+    <string name="quest_address_street_noStreet_confirmation_title">Are you sure that named street is not a part of this address?</string>
+    <string name="quest_address_street_noStreet_confirmation_positive">Without a street part</string>
+    <string name="quest_address_street_place_name_label">Place forming part of the address:</string>
```

Comment:
The wording is a bit awry. Maybe "To what street does the house number X belong"? Etc..

Reply:
I've had a go at improving them. Some of these may need to change as I attempt to get the place names working better too.